test/support/support.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Membrane.VideoMerger.Support do
+  @moduledoc false
+  import Membrane.Testing.Assertions
+  import ExUnit.Assertions
+  alias Membrane.{Buffer, Testing}
+
+  @exact_result_period Membrane.Time.second()
```

Comment:
This is only true for framerates with 1 in the denominator. The (in)famous NTSC framerate (`3000/1001`) won't work with that. Either add a comment or use a number of seconds from the framerate tuple to calculate the period of exact timestamp:
```elixir
iex(5)> Ratio.new(30, 1 |> Membrane.Time.seconds) |> Ratio.denominator |> Membrane.Time.as_milliseconds
100
```
Ratio simplifies the fractions. OFC you'd have to keep it as `Membrane.Time` (so without last call) to use `rem` on expected PTS